sites/public/page_content/locale_overrides/general.json

```diff
@@ -15,5 +15,17 @@
     "headerLink": "City of Detroit - Housing and Revitalization Department",
     "forGeneralInquiries": "For general program inquiries, you may call the City of Detroit HRD at (313) 224-6380.",
     "forListingQuestions": "For listing and application questions, please contact the Property Agent displayed on each listing."
+  },
+  "eligibility": {
+    "progressHeader": "Let's find your next home",
+    "welcome": "Welcome",
+    "welcomeInformation": "To find you a home that best fits your needs, let's answer a few questions first. Some properties only accept people with certain ages, incomes, etc.",
+    "bedroomCountPrompt": "How many bedrooms do you need?",
+    "agePrompt": "How old are you?",
+    "agePromptDescription": "Some properties only accept individuals over 62 years of age.",
+    "ageLabel": "years old",
+    "disabilityPrompt": "Do you have a disability?",
+    "disabilityDescription": "Some properties are meant for residents with disabilities. Some homes might request proof of disability with your application.",
+    "disabilityPreferNotToSay": "Prefer not to say"
   }
 }
```

Comment:
nit: I realize that Exygy has specified `"moreThanThreeBedroom": "3+ Bedroom"`, but does that sound right to us? 🙃  To my mind, "4+ Bedroom" would make more sense, but this is suchhhh a nit.

Reply:
Good call! I added a local definition for these options, but it might be worth updating them in Bloom as well later on.